.github/scripts/launch.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -e
+root=$(cd "$(dirname "$0")/../.." && pwd)
+# get_date return date
+function get_date {
+    backup_dir=$(date +'%Y-%m-%d')
+    echo "$backup_dir"
+}
+
+function usage {
+    echo "./$(basename "$0") -u image_url -l launch_script -f --> show usage"    
+}
+
+# check_gh set default values.
+function check_gh {
+    if [ -z "${IMAGE_URL}" ]; then
+        echo "IMAGE_URL is unset";
+        echo "Set default Values";
+        IMAGE_URL="https://raw.githubusercontent.com/kubernetes-sigs/image-builder/master/images/capi/packer/outscale/ci/nightly"
+    else
+        echo "IMAGE_URL is set to '$IMAGE_URL'"
+    fi
+    IMAGE_TARGET=("centos-7" "ubuntu-2004") 
+    if [ -z "${GIT_FILE_PATH}" ]; then
+        echo "GIT_FILE_PATH is unset"; 
+        echo "Set Default Values";
+        GIT_FILE_PATH=docs/src/topics/omi.md
+    else
+        echo "README is set to '$GIT_FILE_PATH'"
+    fi
+
+    if [ -z "${GIT_CURRENT_BRANCH}" ]; then
+        echo "GIT_CURRENT_BRANCH is unset";
+        echo "Set default Values";
+        GIT_CURRENT_BRANCH=main
+    else
+        echo "GIT_CURRENT_BRANCH is set to '$GIT_CURRENT_BRANCH'"
+    fi
+
+    if [ -z "${LAUNCH_SCRIPT}" ]; then
+        echo "LAUNCH_SCRIPT is unset";
+        echo "Set default Values";
+        LAUNCH_SCRIPT="-o outscale-vbr -r cluster-api-provider-outscale -u ${GIT_CURRENT_BRANCH} -n outscale-vbr -e vincent.baer@outscale.com" 
```

Comment:
we did not define LAUNCH_SCRIPT in the make, right ? so we take this default value with 'vbr'

Reply:
Ok i change the way of work.